daemon/api_aspects.go

```diff
@@ -63,8 +63,8 @@ func getAspect(c *Command, r *http.Request, _ *auth.UserState) Response {
 		var value interface{}
 		err := aspectstateGetAspect(tx, account, bundleName, aspect, field, &value)
 		if err != nil {
-			if errors.Is(err, &aspects.FieldNotFoundError{}) {
-				// keep looking; return partial result, if only some fields are found
+			if errors.Is(err, &aspects.NotFoundError{}) && len(fields) > 1 {
+				// keep looking; return partial result if only some fields are found
```

Comment:
stating the obvious - this partial result behaviour should be clearly documented :)